lib/notifier.js

```diff
@@ -53,6 +58,10 @@ class Notifier {
             notificationWindow.close()
         }, 4000)
 
+        const self = electron.remote && electron.remote.BrowserWindow.getCurrentWindow();
```

Comment:
This is throwing an error, it's `electron.remote.getCurrentWindow()` now I believe.